modules/zscaler/remote-state.tf

```diff
@@ -0,0 +1,19 @@
+module "vpc" {
+  source  = "cloudposse/stack-config/yaml//modules/remote-state"
+  version = "0.17.0"
+
+  stack_config_local_path = "../../../stacks"
+  component               = "vpc"
+
+  context = module.this.context
+}
+
+module "eks" {
```

Comment:
this is specific to customers, maybe it should not be here?

Reply:
I think this is still required for using the sg of the eks in the zscaler ec2 instance